src/flow_solver/flow_solver.cpp

```diff
@@ -337,6 +338,31 @@ void FlowSolver<dim,nstate>::output_restart_files(
 }
 #endif
 
+template <int dim, int nstate>
+void FlowSolver<dim,nstate>::perform_mesh_adaptation() const
```

Comment:
maybe call this `perform_steady_state_mesh_adaptation()` instead based on its definition

Reply:
Fixed.